docs/se/reactivemessaging/introduction.adoc

```diff
@@ -121,13 +132,576 @@ implicit ack before callback is executed is necessary.
 
 === Connector
 Connector concept is a way for connecting <<Channel,channels>> to external sources.
-To make xref:connector.adoc[creation and usage of connectors]
+To make <<Messaging Connector,creation and usage of connectors>>
 as easy and versatile as possible, Helidon SE Messaging uses same API for connectors
 like xref:../../mp/reactivemessaging/introduction.adoc[MicroProfile Reactive Messaging] does.
 This allows connectors to be usable in both flavors of Helidon with one limitation which is
 that connector has to be able to work without CDI.
 
 Example of such a versatile connectors in Helidon:
 
- * xref:kafka.adoc[Kafka connector]
- * xref:jms.adoc[JMS connector]
+* <<Kafka Connector,Kafka connector>>
+* <<JMS Connector,JMS connector>>
+* <<AQ Connector,AQ Connector>>
+
+
+
+== Configuration
+
+* <<Configuration for Messaging connector,Configuration for Messaging connector>>
+* <<Explicit config with config builder for Kafka Connector, Explicit configuration with config builder for Kafka Connector>>
+* <<Implicit Helidon Config for Kafka Connector, Implicit Helidon Configuration for Kafka Connector>>
+* <<Explicit config with config builder for JMS Connector, Explicit configuration with config builder for JMS Connector>>
+* <<Implicit Helidon Config for JMS Connector, Implicit Helidon Configuration for JMS Connector>>
+
+== Reference
+
+* link:{microprofile-reactive-messaging-spec-url}[MicroProfile Reactive Messaging Specification]
+* link:https://github.com/eclipse/microprofile-reactive-messaging[MicroProfile Reactive Messaging on GitHub]
+
+== Examples
+
+* <<Channel,Example of simple channel>>
+* <<Processor,Example of processor usage>>
+* <<Kafka Connector,Kafka connector>>
+* <<JMS Connector,JMS connector>>
+* <<AQ Connector,AQ Connector>>
+
+For a complete example, see link:{helidon-github-tree-url}/examples/messaging[Helidon Messaging Examples].
+
+== Additional Information
```

Comment:
I would remove `Additional Information` and move all connectors under `Connector` section above. Also update ToC accordingly.